config/crd/bases/infrastructure.cluster.x-k8s.io_doclusters.yaml

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
Is this new line needed?

Reply:
addressed, thanks!

Reply:
whops, this is generated file from `controller-gen`, when trying run `make generate-manifests` this new line will be added. so i let it as before.